django_simple_bulma/utils.py

```diff
@@ -0,0 +1,70 @@
+"""Django Simple Bulma utilities. Ultimately helps insure DRY code."""
+
+from pathlib import Path
+from typing import Generator, List, Union
+
+from django.conf import settings
+
+# If BULMA_SETTINGS has not been declared if no extensions
+# have been defined, default to all extensions.
+if hasattr(settings, "BULMA_SETTINGS"):
+    extensions = settings.BULMA_SETTINGS.get("extensions", "_all")
+else:
+    extensions = "_all"
```

Comment:
I'm wondering if we should just remove this bit. It seems like a lot of people are probably going to be loading all the extensions when in fact they're not using any of them, just because they didn't read the readme carefully enough. In fact, I think even the Python Discord website was doing that earlier - it might even still be doing that, maybe I just made an issue to fix it and never actually fixed it.

I'm not sure why I did this in the first place, but it seems silly to default to loading lots of stuff. Who would possibly need every extension?

Reply:
By the way, if you do remove this bit, the README will also need an update. And do feel free to remove it.

Reply:
I agree with this and have committed the changes.

Reply:
Also the pydis site does have all extensions enabled. https://github.com/python-discord/site/blob/master/pydis_site/settings.py#L300-L319